packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -76,27 +81,13 @@ export const upgradeVaults = async (powers, { options }) => {
    *   Installation<import('../../src/vaultFactory/vaultFactory.js')['start']>
    * >}
    */
-  let installationP;
+  const installationP = E(zoe).installBundleID(bundleID);
+  produceVaultInstallation.reset();
+  produceVaultInstallation.resolve(installationP);
 
   await auctionsUpgradeComplete;
   auctionsUpgradeCompleteProducer.reset();
 
-  if (vaultsRef) {
-    if (bundleID) {
-      installationP = E(zoe).installBundleID(bundleID);
-      await E.when(
-        installationP,
-        installation =>
-          E(E(agoricNamesAdmin).lookupAdmin('installation')).update(
```

Comment:
I would expect `agoricNamesAdmin` to disappear from the permit of the auction part. I wonder why I don't see it.

Reply:
`upgradeVaults.js` still uses it for the auction instance at line 226. Should that use promise space instead?

`add-auction.js` added it in `b6e0739`, but dropped it in `37505e3`.

Reply:
> upgradeVaults.js still uses it for the auction instance at line 226. Should that use promise space instead?

yes.

Reply:
done